azure/__init__.py

```diff
@@ -276,6 +276,10 @@ def _to_datetime(strtime):
     'aad_tenant_id': 'AADTenantID',
     'start_ip_address': 'StartIPAddress',
     'end_ip_address': 'EndIPAddress',
+    'operation_id': 'OperationId',
+    'operation_object_id': 'OperationObjectId',
+    'client_ip': 'ClientIP',
+    'id': 'ID'
```

Comment:
This mapping from `id` to `ID` will break the deserialization for several existing model classes that have an `id` field that is currently automatically mapped to `Id`.

If you look in `servicemanagement/__init__.py`, you'll see that there are several.

One possible workaround would be to rename the `id` field on SubscriptionOperationStatus to something unique, like `op_status_id`, and have the mapping be `'op_status_id': 'ID'`.